custom/enikshay/management/commands/base_data_dump.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+import csv
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+DOMAIN = "enikshay"
+
+
+class BaseDataDump(BaseCommand):
+    def __init__(self, *args, **kwargs):
+        super(BaseDataDump, self).__init__(*args, **kwargs)
+        self.log_progress = None
+        self.result_file_name = None
+        self.case_type = None
+        self.input_file_name = None
+        self.report = {}
+        self.result_file_headers = []
+
+    def add_arguments(self, parser):
+        parser.add_argument('case_type')
+        parser.add_argument('input_file_name')
+
+    def setup_result_file_name(self):
+        result_file_name = "data_dumps_{case_type}_{timestamp}.csv".format(
+            case_type=self.case_type,
+            timestamp=datetime.now().strftime("%Y-%m-%d--%H-%M-%S"),
+        )
+        return result_file_name
+
+    def setup(self):
+        with open(self.input_file_name, 'rU') as input_file:
+            reader = csv.DictReader(input_file)
```

Comment:
Since you're transforming the input file anyways, it may be easier to just stick it in the code as a tuple or something so there's no need to upload and parse it.